package.json

```diff
@@ -41,18 +41,22 @@
     "@gitbeaker/node": "30.5.0",
     "@octokit/rest": "18.12.0",
     "@snyk/configstore": "^3.2.0-rc1",
+    "@types/base-64": "^1.0.0",
     "axios": "^0.21.0",
+    "base-64": "^1.0.0",
     "bunyan": "1.8.15",
     "debug": "4.3.2",
     "lodash": "4.17.21",
     "needle": "2.9.1",
+    "nock": "^13.2.1",
     "p-map": "4.0.0",
     "parse-link-header": "1.0.1",
     "sleep-promise": "8.0.1",
     "snyk-config": "^4.0.0",
     "snyk-request-manager": "1.4.1",
     "source-map-support": "^0.5.16",
     "split": "1.0.1",
+    "tslib": "^2.3.1",
```

Comment:
tslib should not be in prod dependencies, it is only used in `dev`, please remove or move to `dev`

Reply:
Will do

Reply:
Done